engine/liveupdate/src/liveupdate.cpp

```diff
@@ -474,13 +433,18 @@ printf("MAWE: StoreManifest: %p  algorithm: %d\n", manifest, (int)algorithm);
         dmResourceArchive::SetNewArchiveIndex(archive_container, new_index, mem_mapped);
     }
 
-    const dmResource::Manifest* GetCurrentManifest()
+    void SetNewManifest(dmResource::Manifest* manifest)
```

Comment:
Since we're creating a new manifest on the fly at runtime, we also need a new way to hook it up with the resource factory.